src/toast/styled-components.js

```diff
@@ -0,0 +1,57 @@
+/*
+Copyright (c) 2018 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+import {styled} from '../styles';
+import {getSvgStyles} from '../icon/styled-components';
+import {KIND} from './constants';
+import type {SharedStylePropsT, KindTypeT} from './types';
+
+function getBackgroundColor(kind: KindTypeT, theme) {
+  return {
+    [KIND.info]: theme.colors.primary500,
+    [KIND.positive]: theme.colors.positive500,
+    [KIND.warning]: theme.colors.warning500,
+    [KIND.negative]: theme.colors.negative500,
+  }[kind];
+}
```

Comment:
For some reason these feel very dark? The orange almost looks brown:

![image](https://user-images.githubusercontent.com/875591/46490598-4b3f8300-c7bd-11e8-936e-8ec2f5b7c000.png)

(they match the "active" button color, not the default button color)

Reply:
Yeah, in the design they use darker (`500`) colors. But I'll double check with Jeff.